### ISSUE TITLE
EfficientNet based pretrained models

### ISSUE THREAD

**Issue by dexception** [deepinsight/insightface#722]
I don't have access to huge gpu machines ... can anyone train the ms1m model on EfficientNet.
https://github.com/lukemelas/EfficientNet-PyTorch

EfficientNet-B1 gives the same accuracy on Imagenet as Resnet152 with 10 times less computation power.  Worth a try !

**Comment by dexception** [comment]
@nttstar Do you have any plans to implement this ?

**Comment by dexception** [comment]
@nttstar 
Can you this implementation of EfficientNet ?

https://github.com/mnikitin/EfficientNet

**Comment by dexception** [comment]
...

**Comment by feifei9099** [comment]
> @nttstar
> Can you this implementation of EfficientNet ? This is in Mxnet.
> 
> https://github.com/mnikitin/EfficientNet

@dexception 
Have you used this efficientnet yet, how is the performance?

**Comment by deimsdeutsch** [comment]
EfficientNet-B0 is about 5 times faster than Resnet50 and at the same time more accurate than Resnet50.

**Comment by nttstar** [comment]
You can try it, with modifications for face 112x112 input. And I will probably provide some new network backbones in a couple of days.

**Comment by Neltherion** [comment]
@nttstar That'd be great... waiting for them to see if the accuracy could go even higher for MegaFace.

**Comment by dexception** [comment]
@nttstar 
Any updates ?
